tests/TestVersion.py

```diff
@@ -65,6 +65,20 @@ def test_versionPostfix():
     assert version <= Version("1.2.3-alpha.5")
     assert version < Version("1.2.3-alpha.5")
 
+def test_postfix_lastest_json_format():
```

Comment:
- What happens if the pre-release-tag version doesn't contain and number `<pre_release_tag>.1` like it does now?
- What happens with the build meta data after the `+`